scripts/laserscan_to_distance.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+import rospy
+import time
+import numpy as np
+from std_msgs.msg import Float64
+from sensor_msgs.msg import LaserScan
+from robotont_msgs.msg import LaserScanSplit
+
+pub = rospy.Publisher('/scan_to_distance', LaserScanSplit, queue_size=1)
```

Comment:
Let's avoid absolute paths in nodes, so this file can be used in multiple namespaces when necessary.
```suggestion
pub = rospy.Publisher('scan_to_distance', LaserScanSplit, queue_size=1)
```

Check also other occurrences in this PR.